console/urls.py

```diff
@@ -158,6 +159,8 @@
     url(r'^users/query$', UserFuzSerView.as_view()),
     url(r"^users/access-token$", UserAccessTokenCLView.as_view()),
     url(r"^users/access-token/(?P<id>[\w\-]+)$", UserAccessTokenRUDView.as_view()),
+    url(r'^monitor/metrics$', MonitorMetricsView.as_view()),
```

Comment:
Should be obtained from the component model.

Reply:
fixed